syntax = "proto3";

package cilium;

message BpfMetadata {
  // File system root for bpf. Defaults to "/sys/fs/bpf" if left empty.
  string bpf_root = 1;

  // 'true' if the filter is on ingress listener, 'false' for egress listener.
  bool is_ingress = 2;
}
